docker-compose.yml

```diff
@@ -53,18 +53,28 @@ services:
       - ${HOME}/ckan_cache:/home/netkan/ckan_cache
   webhooks:
     build:
-      context: webhooks/.
+      context: netkan/.
+      target: dev
```

Comment:
I think this `target` can be left out since the dockerfile doesn't use multi-stage builds.

Reply:
OK, I don't quite understand what that line does, but I'll take your word for this.

Reply:
If you look at netkan/Dockerfile, you see those FROM .. as ... lines.
These are for [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/).
So we first create a base image, and build different images for dev and production on it, where the dev image is "tagged" as `dev`...
https://github.com/KSP-CKAN/NetKAN-Infra/blob/19241c3c465d839d7e98f31f1fdfa77780cbc749/netkan/Dockerfile#L12-L19

...and the other one has no stage tag, which means it is the default one.
https://github.com/KSP-CKAN/NetKAN-Infra/blob/19241c3c465d839d7e98f31f1fdfa77780cbc749/netkan/Dockerfile#L21-L29

If you reference a Dockerfile in docker-compose to build from, without specifying a target the default one will be used.
So we need to specify `target: dev` to tell docker-compose to build the `dev` stage.

Hopefully this is somewhat understandable.

Reply:
Ohhhh!
So it just skips that final block that makes it print the help text when you start without parameters?
I thought it had something to do with the `dev` block later in this same file!

https://github.com/KSP-CKAN/NetKAN-Infra/blob/0e09894cb8b15af58d0d03c0ed24b1331e905802/docker-compose.yml#L130-L141

Reply:
Plus the final block doesn't run the `pip install --user -e netkan/.` command, but yes.
I'm not sure what the `dev` block of the compose is for either.

Reply:
What about `dev-stack.py`? I suppose that needs an update as well?